docs/grammar/SolidityLexer.g4

```diff
@@ -234,7 +234,7 @@ fragment IdentifierStart: [a-zA-Z$_];
 //@doc:inline
 fragment IdentifierPart: [a-zA-Z0-9$_];
 
-WS: [ \t\r\n\u000C]+ -> skip ;
```

Comment:
This here is actually rather about whitespace in the actual source file, not about escape sequences. And actually I just took this part over from the previous grammar without thinking about it too much, so in general it may be worth a look how source files with such whitespace in them behaves.

The actual escape sequences are defined in ``fragment EscapeSequence``, where I just put in all of them allowed by the scanner.

Reply:
Right, that makes sense, this commit was done some evening a week ago :)

Reply:
Actually `test/liblanglutil/Scanner` has some tests regarding line endings -- that could be used to review the `LINE_COMMENT` lexer item.